core/vulkan/vk_virtual_swapchain/cc/BUILD.bazel

```diff
@@ -1,5 +1,7 @@
 load("//tools/build:rules.bzl", "cc_copts", "cc_defines")
 
+exports_files(["VirtualSwapchainLayer.json"])
```

Comment:
It's really rare for exports_files to be a better answer than a filegroup with a single file